features/ownership/get-creator.feature

```diff
@@ -0,0 +1,47 @@
+Feature: Test getting creator ownership
+    Background:
+        Given I am using the UDB3 base URL
+        And I am using an UiTID v1 API key of consumer "uitdatabank"
+        And I send and accept "application/json"
+
+    Scenario: Getting the creator of an ownership as creator
+        Given I am authorized as JWT provider v2 user "dev_e2e_test"
```

Comment:
I think this can move to the background because it seems to be the same for all 3 scenarios

Reply:
[Move shared auth step to background](https://github.com/cultuurnet/udb3-backend/pull/1863/commits/ce9b5ecadcd451e41852e0022a259dcf16430f9f)